friends/src/components/FriendList.js

```diff
@@ -0,0 +1,38 @@
+import React, { Component } from "react";
+import { connect } from "react-redux";
+import { fetch } from '../actions';
+import Friend from './Friend';
+
+class FriendList extends Component {
+    constructor() {
+        super();
+    }
+
+    componentDidMount() {
+        this.props.fetch()
+    }
+
+    render() {
+        if (this.props.fetchingFriends) {
+            return <div className='loading'>Friends loading...</div>
```

Comment:
```suggestion
            return <div className='loading'>Friends loading...</div>
```
eternal sadness ... 😞